proposals/NNNN-swift-openapi-generator.md

```diff
@@ -0,0 +1,120 @@
+# Swift OpenAPI Generator
+
+* Proposal: [SSWG-NNNN](NNNN-swift-openapi-generator.md)
+* Authors: [Honza Dvorsky](https://github.com/czechboy0)
+* Review Manager: [Joannis Orlandos](https://github.com/joannis)
+* Status: **Implemented**
+* Implementation: [apple/swift-openapi-generator](https://github.com/apple/swift-openapi-generator)
+* Forum Threads: [Pitch](https://forums.swift.org/t/pitch-sswg-incubation-swift-openapi-generator/66584), [Discussion](https://forums.swift.org/), [Review](https://forums.swift.org/)
+
+*During the review process, add the following fields as needed:*
+
+* Decision Notes: [Rationale](https://forums.swift.org/), [Additional Commentary](https://forums.swift.org/)
+* Previous Revision(s): [1](https://github.com/swift-server/sswg/blob/...commit-ID.../proposals/NNNN-filename.md)
+* Previous Proposal(s): [SSWG-XXXX](XXXX-filename.md)
```

Comment:
@czechboy0 that was supposed to be tagged to these lines

Reply:
Removed: https://github.com/swift-server/sswg/pull/86/commits/cf500b30b85419ea4bc838d93325a07f8a4007d2